src/sentry/replays/endpoints/organization_replay_details.py

```diff
@@ -58,6 +58,12 @@ def get(self, request: Request, organization: Organization, replay_id: str) -> R
         except NoProjects:
             return Response(status=404)
 
+        try:
+            assert filter_params["start"]
+            assert filter_params["end"]
+        except Exception:
```

Comment:
Not sure what our policy is on using `assert` for control flow, but the general advice is not to AFAIK.

```suggestion
        if not (filter_params.get("start") and filter_params.get("end")):
```

Reply:
👍 cool ya makes sense